src/index.html

```diff
@@ -11,7 +11,139 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <div class="container">
+      <form class="form"
+      action="https://github.com/mate-academy/layout_html-form/blob/master/checklist.md"
+      method="post"
+      >
+        <fieldset class="form__personal blockStyle">
+          <legend>Personal information</legend>
+          <div class="form__personal-surname mBottom">
+            Surname:
+            <input name="surname"
+              type="text"
+              autocomplete="off"
+            >
+          </div>
+          <div class="form__personal-name mBottom" required>
```

Comment:
Please remove unnecessary classes. Please use only two for Vertical distance between inputs(10px) and vertical distance between groups(20px). Also check for all cases